TESTS/netsocket/dns/dns_tests.h

```diff
@@ -19,10 +19,14 @@
 #define DNS_TESTS_H
 
 #ifndef MBED_CONF_APP_DNS_SIMULT_QUERIES
+#ifdef MBED_CONF_CELLULAR_OFFLOAD_DNS_QUERIES
+#define MBED_CONF_APP_DNS_SIMULT_QUERIES   MBED_CONF_CELLULAR_OFFLOAD_DNS_QUERIES
+#else
 #define MBED_CONF_APP_DNS_SIMULT_QUERIES   5
 #endif
+#endif
 
-#ifndef MBED_CONF_NSAPI_DNS_CACHE_SIZE
+#ifdef MBED_CONF_CELLULAR_OFFLOAD_DNS_QUERIES
```

Comment:
I think this should also obey MBED_CONF_NSAPI_DNS_CACHE_SIZE setting

Reply:
You think right, fixed.